google-feeds.html

```diff
@@ -30,26 +30,30 @@
     is: "google-feeds",
 
     /**
-     Fired when feed has been loaded
-     @param {object} feed feed object
-     @event google-feeds-response
-
-     Fired when feed load fails
-     @param {string} status load status
-     @event google-feeds-error
-
-     Fired when multiple feeds have loaded successfully
-     @param {object} feeds multiple feeds
-     @event google-multi-feeds-response
-
-     Fired when feed query fails
-     @param {string} status status
-     @event 'google-feeds-queryerror'
-
-     Fired when feed query succeeds
-     @param {object} entries query results
-     @event 'google-feeds-queryresponse'
-    */
+     * Fired when feed has been loaded
```

Comment:
Yep. The way parser works is that it uses all free text within the block as description, and all @tags as properties of that comment. 

Aleks
-- sent from my phone

> On May 13, 2015, at 4:17 PM, Eric Bidelman <notifications@github.com> wrote:
> 
> In google-feeds.html:
> 
> > -     @param {string} status load status
> > -     @event google-feeds-error
> > -
> > -     Fired when multiple feeds have loaded successfully
> > -     @param {object} feeds multiple feeds
> > -     @event google-multi-feeds-response
> > -
> > -     Fired when feed query fails
> > -     @param {string} status status
> > -     @event 'google-feeds-queryerror'
> > -
> > -     Fired when feed query succeeds
> > -     @param {object} entries query results
> > -     @event 'google-feeds-queryresponse'
> > -    */
> > +     * Fired when feed has been loaded
> These all need to separate comment blocks? That's a weird requirement.
> 
> —
> Reply to this email directly or view it on GitHub.
>